idea_plugin/src/com/google/googlejavaformat/intellij/InitialConfigurationProjectManagerListener.java

```diff
@@ -38,6 +40,9 @@ public void projectOpened(@NotNull Project project) {
     if (settings.isUninitialized()) {
       settings.setEnabled(false);
       displayNewUserNotification(project, settings);
+
+      // disable auto-updates of the plugin
+      UpdateChecker.getDisabledToUpdate().add(PluginId.getId("kuzyanov_google-java-format"));
```

Comment:
https://intellij-support.jetbrains.com/hc/en-us/community/posts/206756405-How-to-disable-plugin-auto-update-programmatically-